setup.py

```diff
@@ -63,6 +63,20 @@ def main():
         ],
     )
 
+    # if this is being set up in a git repo
+    if glob('.git'):
+        enforce_pep8()
+
+def enforce_pep8():
```

Comment:
it's a cheaty but easy way to check that this is a developer environment using git.